accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -357,6 +420,7 @@ public static void main(String[] args)
     @Test
     public void testOne()
     {
+//        run(726880888609333L, 1000); // taking a long time...  gave up after 15 minutes!
```

Comment:
will revert, this is just a seed that hit a while true loop in recovery. Spoke to Benedict and seems that his branch may fix this